Automattic/wp-calypso
# Purchases: Prettify all files with prettier!

This is an experiment to validate `prettier` usage which was enabled with #14010. I would love to hear your feedback about this change. Reader team has already done that and they happily use `codemods` :)

Once we have all files styled with `prettier`, we should have much simplified flow when using `codemods`. Every time you execute `codemod`, you run prettier again to fix all spacing or new lines issues. See #13965 to find out how Reader team is taking advantage of `prettier`.

Prettier in action with PhpStorm:

![prettier](https://cloud.githubusercontent.com/assets/699132/26094165/1bfb960c-3a19-11e7-9815-cf10647b180a.gif)

More details how to setup with Jetbrains: https://github.com/prettier/prettier/tree/master/editors/jetbrains.

Pretttier from CLI:

```
./node_modules/.bin/prettier --write client/me/purchases/**/*.js*
```

There is no change in logic with this PR. It's mostly about introducing code style with prettier. I also fixed 2 linter issues.

Note: there are still errors triggered by Eslint on CircleCI, all of them related to missing `localize` usage. They will be relaxed to warnings on master, so we can leave them for another PR.

### Testing

1. Open  https://calypso.live/me/purchase?branch=update/prettify-me-purchases.
2. Make sure it works as before.
